Assets/Scripts/GameScene/BoardGenerator.cs

```diff
@@ -1,38 +1,50 @@
-using System.Collections;
-using System.Collections.Generic;
+using System;
 using UnityEngine;
 
 public class BoardGenerator : MonoBehaviour
 {
-    [SerializeField] private float width;
-    [SerializeField] private float height;
-    [SerializeField] private FloorTile tile;
-    
-    
-    // Start is called before the first frame update
-    void Start()
+    public static event Action<BoardGenerator> BoardDidLoad;
+    [SerializeField] private Board board; 
+    public Vector3[,] boardData { get; private set; }
```

Comment:
wtf is this? why does it has a public getter and private setter? who accesses this?

Reply:
Took it from the DigDag replica because we want that only the getter will be public. 
We don't want other classes to change the board coordinations after initialization.